tests/test_values.py

```diff
@@ -443,13 +439,7 @@ def test_cache_url_value(self):
         with env(CACHE_URL='wrong://user@host:port/1'):
             with self.assertRaises(Exception) as cm:
                 value.setup('TEST')
-            self.assertEqual(cm.exception.args[0], 'Unknown backend: "wrong"')
-        with env(CACHE_URL='redis://user@host:port/1'):
-            with self.assertRaises(ValueError) as cm:
-                value.setup('TEST')
-            self.assertEqual(
-                cm.exception.args[0],
-                "Cannot interpret cache URL value 'redis://user@host:port/1'")
+            self.assertEqual(cm.exception.args[0], 'wrong')
```

Comment:
That looks "wrong"?!
Shouldn't it have more info / a full msg?

Reply:
django-environ simply raises `KeyError` when looking up a cache scheme by `wrong`, and here `assertRaises` catches the `KeyError`.

It might be good for django-envion to catch the KeyError and reraise a specific error with more descriptive message.

What do you think?

Reply:
FYI, I made a patch to django-environ, https://github.com/joke2k/django-environ/pull/235

Reply:
@tkdchen , that patch was merged; I guess this can be updated now ?